bin/faucet/src/backend/get_metadata.rs

```diff
@@ -12,6 +12,7 @@ use url::Url;
 use crate::COMPONENT;
 
 /// Describes the faucet metadata needed to show on the frontend.
+#[derive(Clone)]
 pub struct Metadata {
```

Comment:
minor nit: Not from this PR but feels like `Metadata` is too general. Not sure what a good alternative would be though, since `FaucetMetadata` is a bit misleading considering it also has the explorer URL